jboss-container-images/redhat-sso-7-openshift-image
# [KEYCLOAK-17694] Fix the 'jboss' username mapping to work properly also for dynamically generated container's user IDs

<hr/>

    [KEYCLOAK-17694] Fix the 'jboss' username mapping to work
    properly also for dynamically generated container's user IDs
    
    Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>
<hr/>

_Testing instructions:_

1. Deploy the image on OpenShift & wait for the pod to be running:
```
$ oc get pods
NAME          READY   STATUS    RESTARTS   AGE
sso-1-vx4sl   1/1     Running   0          39m
```
2. `rsh` to the pod in question & issue the `whoami` command to confirm the dynamically generated user ID the container is run with (`$(id -u)`) got properly mapped to `jboss` username:
```
$ oc rsh sso-1-vx4sl whoami
jboss
```

```
$ oc rsh sso-1-vx4sl id -u
1000080000
```

```
$ oc rsh sso-1-vx4sl id
uid=1000080000(jboss) gid=0(root) groups=0(root),1000080000
```
<hr/>
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`
